packages/esbuild/test/index.js

```diff
@@ -414,7 +415,8 @@ test('@mdx-js/esbuild', async function (t) {
         file.message('3', tree)
         file.message('4', esm)
         file.message('5', text)
-        file.message('6', jsx)
+        const m6 = file.message('6', jsx)
+        if (m6.place && 'start' in m6.place) delete m6.place.start.offset
```

Comment:
```suggestion
        assert(m6.place)
        assert('start' in m6.place)
        delete m6.place.start.offset
```

Use assertions so this doesn’t accidentally silently fall through and test something else